README.md

```diff
@@ -2,58 +2,6 @@
 
 This hosts multiple scripts necessary for filtering and processing of variant calls in the vcfs/txt file generated by callers.
 
-## Callers Supported
-`pv` is the main command for the `postprocessing_variant_calls` package see `pv --help` to see supported variant callers commands. 
-
-### VarDictJava
-
-The sub-command `pv vardict` allows users to perform post-processing on VarDictJava output. The two supported inputs to `pv vardict` from VarDictJava are `single` and `case-control` vcfs. 
```

Comment:
very minor change, but I'd recommend adding the version of VarDictJava in the README. mostly due to the fact that the docker image created for VarDict that is compatible with M1 is version 1.8.2 (same version used by MSK), but the version used by nf-core is 1.8.3

Reply:
good call. I added the version in the `docs/VARDICT.md` and added a link in the README.